spec.bs

```diff
@@ -468,6 +515,19 @@ try {
     [[webauthn-conditional-ui inline]]. It is likely that both it and SPC could
     be supported by the same underlying API.
 
+: <dfn>Browser bound key</dfn>
+:: A public-private key pair that signs over the transaction in addition to the
+    WebAuthn credential and is tied to a single device by the user agent.
+
+: <dfn>Platform Cryptographic APIs</dfn>
```

Comment:
This is never referenced in the text; is it valuable to have it defined here? Notably, I don't think WebAuthn defines something like this, although perhaps this is because they have the [Authenticator model](https://w3c.github.io/webauthn/#sctn-authenticator-model) instead (which may be useful as a template for some of this work!)